src/Symfony/Component/Lock/Store/DoctrineDbalStore.php

```diff
@@ -235,4 +248,23 @@ private function getCurrentTimestampStatement(): string
                 return (string) time();
         }
     }
+
+    /**
+     * Checks wether current platform supports table creation within transaction.
```

Comment:
can be removed, the name of the method says it already

Reply:
I've kept this comment as this class is full of such comments :)